<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormInfoGeneral</class>
 <widget class="QWidget" name="FormInfoGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>385</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBoxFont">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblFontHandle">
        <property name="text">
         <string>Handle:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txtFontHandle">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="lblFontName">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="2">
       <widget class="QLineEdit" name="txtFontName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2">
       <widget class="QLineEdit" name="txtFontSize">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lblFontSize">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblUnicode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Is unicode:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QLineEdit" name="txtUnicode">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="lblWindowBytes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Window bytes:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QComboBox" name="cboWindowBytes"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="lblUserData">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>User data:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLineEdit" name="txtUserData">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="lblControlId">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Control id:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="txtControlId">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lblMenu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Menu handle:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLineEdit" name="txtMenu">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblInstance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Instance handle:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLineEdit" name="txtInstance">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblProcedure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Window procedure:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="txtProcedure">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lblWindowText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Window text:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="cboWindowText">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lblHandle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Handle:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnHandle">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../res/resource.qrc">
       <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="txtHandle">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnHandle</sender>
   <signal>clicked()</signal>
   <receiver>FormInfoGeneral</receiver>
   <slot>slotWindowHandle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotWindowHandle()</slot>
 </slots>
</ui>
